cg/apps/sequencing_metrics_parser/models.py

```diff
@@ -0,0 +1,22 @@
+from pydantic import BaseModel, Field
```

Comment:
moved and renamed models from `cg/apps/sequencing_metrics_parser/models/bcl_convert.py`